app/helpers/users_helper.rb

```diff
@@ -6,4 +6,48 @@ def owner_image_tag(size=nil)
   def owner_image_link
     person_image_link(current_user.person, :size => :thumb_small)
   end
+
+  # Returns the path of the current color theme so that it
+  # can be loaded in app/views/layouts/application.html.haml
+  # and app/views/layouts/application.mobile.haml. If the user
+  # is not signed in or has not specified a color theme, the
+  # default (original) color theme is loaded.
+  #
+  # ==== Examples
+  #
+  #   # if user is not signed in
+  #   current_color_theme # => "color_themes/original"
+  #   # if user Alice has not selected a color theme
+  #   current_color_theme # => "color_themes/original"
+  #   # if user Alice has selected a "magenta" theme
+  #   current_color_theme # => "color_themes/magenta"
+  def current_color_theme
+    if user_signed_in?
+      color_theme = current_user.color_theme
+    end
+    color_theme ||= DEFAULT_COLOR_THEME
```

Comment:
@jhass if I get it correctly, this PR would allow each user to set the color theme they want to. So it's not a podmin setting.